PolinaProject/Program.cs

```diff
@@ -82,7 +83,11 @@
 
 Console.Write("Task 8. Write any number: ");
 double variableA = double.Parse(Console.ReadLine());
-if ((variableA < 50 && variableA != 37 && variableA > 32) || variableA == 0 || variableA == 15)
+if (variableA < 50 && variableA != 37 && variableA > 32)
+{
+    Console.WriteLine($"Working");
+}
+else if (variableA == 0 || variableA == 15)
```

Comment:
here it's better to revert this change to avoid using code repeating. We don't want to repeat the same code parts as Console.WriteLine($"Working") for example